tests/portforward_test.go

```diff
@@ -0,0 +1,96 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package tests_test
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+var _ = Describe("[sig-compute]PortForward", func() {
+
+	var err error
+	var virtClient kubecli.KubevirtClient
+
+	var (
+		LaunchVMI func(*v1.VirtualMachineInstance) *v1.VirtualMachineInstance
+	)
+
+	tests.BeforeAll(func() {
```

Comment:
Please don't use `BeforeAll`. We can not support it properly. Just use `BeforeEach`. We should add a `Deprecated` description on the function.